src/components/InstitutionForm.svelte

```diff
@@ -0,0 +1,136 @@
+    }
+    };
+    
+    const buildLocation = () => {
+        if (inAmerica)
+            {
+                return cityName + ", " + stateName;
+            }
+            else
+            {
+                return cityName + ", " + countryName;
+            }
+    };
+
+    const validateSubmission = () => {
+        let valid = true;
+        let errorMessage = "ERROR! The following fields are required: "
+        if (instName == "" || instName == undefined)
+        {
+            valid = false;
+            errorMessage += "\n-Institution Name"
+        }
+        if (stateName == "" || cityName == undefined)
+        {
+            valid = false;
+            errorMessage += "\n-City"
+        }
+        if (stateName == "" && (countryName == undefined || countryName == ""))
+        {
+            valid = false;
+            errorMessage += "\n-State or Country"
+        }
+        if (!valid)
+        {
+            alert(errorMessage);
+        }
+        return valid;
+    };
+
+</script>
+<div class = "inside">
+<h1 class = "formType">New Non-Academic Insitution</h1>
+<h2>1. Enter Information</h2>
+<p><span class = "red">*</span>Required fields</p>
+<form>
+    <p>Name<span class = "red">*</span></p>
+    <input type = "text" placeholder = "Insitution Name" bind:value = {instName} required>
+    
+    <p>Location<span class = "red">*</span></p>
+    <label>
+    <input type="checkbox" bind:checked={inAmerica}/>
+    My institution is in the United States
+</label>
+<br/>
+<br/>
+    {#if inAmerica}
+    <input type = "text" placeholder = "City (ex. New York)" bind:value = {cityName} required>
+        <select bind:value = {stateName}>
+            {#each stateAbbrev as state}
+                <option value={state}>
+                    {state}
+                </option>
+            {/each}
+        </select>
+    {:else}
+    <input type = "text" placeholder = "City (ex. London)" bind:value = {cityName} required>
+    <input type = "text" placeholder = "Country" bind:value = {countryName} required>
+    {/if}
+    <p>Website</p>
+    <input type = "text" placeholder = "Please enter a url" bind:value = {website}>
+    <p>Add tags</p>
+    <InstitutionTag/>
+    <p>Image</p>
+<input accept="image/png, image/jpeg" type="file" bind:files={photo} />
+<p>Description</p>
+    <textarea class = "fill" type = "text" placeholder = "Add additional description here" bind:value = {description}/>
+    <br/>
+    <br/>
+    <button on:click|preventDefault = {handleSubmit}>Next</button>
+</form>
+</div>
+
+<style>
+.inside {
+    width: 80%;
+    border: 2px solid black;
+    height: auto;
+    padding: 2%;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 20px;
+    background-color: white;
+    opacity: 0.92;
+    }
+    .red{
+        color: red;
+    }
+    .fill {
+        overflow-y:auto;
+        height: 80px;
+        width: 100%;
+    }
+    .formType
+    {
+        text-align: center;
+    }
+    </style>
```

Comment:
Would it be possible to centralize some of these repeated CSS classes and import them, rather than repeat them across the different components? Or is that considered bad form from a JS-first approach to web design?